spec.json

```diff
@@ -344,6 +346,7 @@
         },
         "largeMotor": {
             "friendlyName": "Large Motor",
+            "systemDeviceNameConvention": "motor{0}",
```

Comment:
I say we normalize to `*` -- it's a universally understood symbol for a wildcard and single characters are simpler in general.